.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Update Package Properties
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRJ_NAME: ${{ github.event.inputs.project_name }}
+  TAG_NAME: ${{ github.event.inputs.tag_name }}
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+  FANCY: ${{ github.event.inputs.fancy }}
+  FANCY_VERSION_NO: ${{ github.event.inputs.fancy_version_no }}
+  MICROSOFT_EMAIL: ${{ github.event.inputs.microsoft_email }}
+  NAME: ${{ github.event.inputs.name }}
+
+# Controls when the action will run.
+on:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
```

Comment:
Some general question about this PR.
1. Do I understand correctly that we will manually trigger a build of this workflow (by clicking in the interface) to create a PR for a new package version?
2. What is the repo layout of that we want in the future? Still one branch per project? If so, then I think having most of these values be configurable is not very useful. They can be hardcoded in the workflow since each workflow will have a different template.

Reply:
Answers
1. Yes it is true
2. Yes I don't think to change it unless there is a better solution for packaging. Our pipelines are not configurable. Only getting parameters from users. For example in this pipeline project_name, tag_name, project_version, fancy and fancy_version, microsoft_email and name is being requested from user. Git hub token is a secret value so it is being stored on and referenced from Repository Secret Store. In summary, we can not hardcode parameters because most of the parameters are being requested from user and the other one is already hardcoded but hardoced on Github Repository Secret Store

Reply:
@hanefi suggested me to hardcode project_name since project_name is not dynamic on this branch

Reply:
I think `fancy` and `fancy_version` should also be hardcoded, because they should stay the same for the same project.

Reply:
@hanefi what do you think of hardcoding Fancy parameter? I understood from your comments that fancy versioning is not needed all the times. It is useful when there is a need for adding a new release without changing version number

Reply:
It makes sense to have a hardcoded `fancy` value for projects. And a `fancy_version` parameter.

As far as I recall, all our extensions use fancy versioning today, and it is unlikely that we will change that in the near future.